content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -45,24 +42,28 @@ on **one** Kubernetes cluster.
 
 ## Install the Istio remote on every remote cluster
 
-The istio-remote component must be deployed to each remote Kubernetes
-cluster.  There are two approaches to installing the remote.  The remote
-can be installed and managed entirely by Helm and Tiller, or via Helm and
-`kubectl`.
+You must deploy the `istio-remote` component to each remote Kubernetes
+cluster. You can install the component in one of two ways:
 
-### Set environment variables for Pod IPs from Istio control plane needed by remote
+* [Option 1: Use Helm and `kubectl` to install and manage the remote cluster](#kubectl)
```

Comment:
For Option 1/2 kind of thing, I suggest switching to using tabs which I think makes for a cleaner page.

https://preliminary.istio.io/about/contribute/writing-a-new-topic/#using-tabs

Reply:
Thanks for the tip. I will implement the taps.

Reply:
Done